lib/bot.js

```diff
@@ -152,6 +153,11 @@ class Bot {
       return;
     }
 
+    if (message.user === 'USLACKBOT' && this.muteSlackbot) {
```

Comment:
Good point. Would you prefer the change as a separate commit, or do you want me to squash and force-push?